proto/cosmos/vesting/v1beta1/tx.proto

```diff
@@ -28,4 +35,47 @@ message MsgCreateVestingAccount {
 }
 
 // MsgCreateVestingAccountResponse defines the Msg/CreateVestingAccount response type.
-message MsgCreateVestingAccountResponse {}
\ No newline at end of file
+message MsgCreateVestingAccountResponse {}
+
+// MsgCreateClawbackVestingAccount defines a message that enables creating a ClawbackVestingAccount.
+message MsgCreateClawbackVestingAccount {
+  option (gogoproto.equal) = false;
+
+  // from_address specifies the account to provide the funds and sign the
+  // clawback request
+  string from_address = 1;
+  // to_address specifies the account to receive the funds
+  string to_address = 2;
+  // start_time defines the time at which the vesting period begins
+  google.protobuf.Timestamp start_time = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+  // lockup_periods defines the unlocking schedule relative to the start_time
+  repeated cosmos.vesting.v1beta1.Period lockup_periods = 4 [(gogoproto.nullable) = false];
```

Comment:
I wonder what the difference is between vesting and unlocking. They seem the same to me.

Reply:
So the term locking & unlocking stands for cliffs here